src/http/one/TeChunkedParser.h

```diff
@@ -45,10 +45,11 @@ class TeChunkedParser : public Http1::Parser
     virtual Parser::size_type firstLineSize() const {return 0;} // has no meaning with multiple chunks
 
 private:
-    bool parseChunkSize(Http1::Tokenizer &tok);
-    bool parseChunkExtension(Http1::Tokenizer &tok, bool skipKnown);
-    bool parseChunkBody(Http1::Tokenizer &tok);
-    bool parseChunkEnd(Http1::Tokenizer &tok);
+    bool parseChunkSize(Tokenizer &tok);
+    bool parseChunkExtension(Tokenizer &tok, const bool skipKnown);
```

Comment:
Please rename to minimize (missing/extra `s`) typo danger:
```suggestion
    bool parseOneChunkExtension(Tokenizer &tok, const bool skipKnown);
```

Reply:
Done.